HW-0.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 title: "HW-0"
-author: "WRITE YOUR NAME HERE"
-date: "August 15, 2016"
+author: "TRISHA SINGH"
+date: "September 15, 2016"
```

Comment:
GitHub as a great GUI (graphical user interface) for evaluating differences in code. This is how I'll be primarily delivering feedback this semester. Also, you can write comments in Markdown, which R Markdown uses. More on Markdown later.